recipes-extended/jailhouse/jailhouse/0002-scripts-include.mk-multilib-Fix-hardcoded-lib-in-fir.patch

```diff
@@ -0,0 +1,28 @@
+From 5d81cd4eeffb5f8a9e73ee0faf374e72cec0d43c Mon Sep 17 00:00:00 2001
+From: Cristinel Panfir <cristinel.panfir@nxp.com>
+Date: Sat, 14 Mar 2020 01:22:05 +0200
+Subject: [PATCH] scripts/include.mk: multilib: Fix hardcoded lib in
+ firmwaredir
+
+Signed-off-by: Cristinel Panfir <cristinel.panfir@nxp.com>
```

Comment:
This patch is not need; pass `firmwaredir=$(base_libdir)/firmware`  in `EXTRA_OEMAKE` and it should do the trick.

Reply:
Yes, it make sens, I'll discard that patch for now.
I'll check later the passing via `EXTRA_OEMAKE` and if there needs something else added in recipe